siliconcompiler/core.py

```diff
@@ -1925,19 +1930,30 @@ def _read_manifest(self, filename, job=None, clear=True, clobber=True, partial=F
         partial (bool): If True, perform a partial merge, only merging keypaths
         that may have been updated during run().
         """
-        abspath = os.path.abspath(filename)
-        self.logger.debug('Reading manifest %s', abspath)
+
+        filepath = os.path.abspath(filename)
+        self.logger.debug(f"Reading manifest {filepath}")
+        if not os.path.isfile(filepath):
+            self.logger.error(f"Manifest file not found {filepath}")
+            sys.exit()
```

Comment:
Consider `raise SiliconCompilerError(message)` instead of `sys.exit()`? (Or use `sys.exit(1)`?)